client/src/app/+signup/shared/signup-success.component.html

```diff
@@ -11,6 +11,6 @@
   <p>{{ message }}</p>
 
   <p i18n>
-    If you need help to use PeerTube, you can take a look to the <a href="https://docs.joinpeertube.org/#/use-setup-account" target="_blank" rel="noopener noreferrer">documentation</a>.
+    If you need help to use PeerTube, you can take a look on the <a href="https://docs.joinpeertube.org/#/use-setup-account" target="_blank" rel="noopener noreferrer">documentation</a>.
```

Comment:
It's "take a look at". In this case I would prefer "you can have a look at the documentation."

Reply:
Thanks! It's been corrected in the latest commit.